DNN Platform/DotNetNuke.Web.Razor/DotNetNukeWebPage.cs

```diff
@@ -69,7 +69,7 @@ public DotNetNukeWebPage()
         [Obsolete("Deprecated in 9.3.2, will be removed in 11.0.0, use Razor Pages instead")]
         public new TModel Model
         {
-            get { return _model ?? (_model = PageContext.Model as TModel); }
+            get { return PageContext?.Model as TModel ?? _model; }
```

Comment:
Should we add any comments to this file explaining this change or is it okay to leave the history in the PR as documented.

Reply:
This is a deprecated method that is not going to be supported, I think its ok.  But wouldn't argue either.

Reply:
Let's leave the history here in the PR. I think we have enough documentation for us to reference in the future